support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -446,10 +447,27 @@ function CheckoutComponent({ geoId }: Props) {
 		useState(true);
 
 	const [billingPostcode, setBillingPostcode] = useState('');
+	const [billingPostcodeError, setBillingPostcodeError] = useState<string>();
 	const [billingLineOne, setBillingLineOne] = useState('');
 	const [billingLineTwo, setBillingLineTwo] = useState('');
 	const [billingCity, setBillingCity] = useState('');
-	const [billingState, setBillingState] = useState('');
+	const [billingStateError, setBillingStateError] = useState<string>();
+	/**
+	 * BillingState selector initialised to undefined to hide
+	 * billingStateError message. formOnSubmit checks and converts to
+	 * empty string to display billingStateError message.
+	 */
+	const [billingState, setBillingState] = useState<string | undefined>(
+		undefined,
+	);
+	useEffect(() => {
+		if (billingState === '') {
+			setBillingStateError('Please enter a state, province or territory');
+		} else {
+			setBillingStateError('');
+		}
+	}, [billingState]);
```

Comment:
This makes sense - just validating - we're doing this as we don't want to write the validation into the component as it would the double validate in other areas that we use it?

Reply:
When form loads, `billingState = undefined` so don't show the error message. If you have submitted or pressed the drop down and not seleceted a state, then it will display error message.